### DISCUSSION TITLE
Fixes extension detection

### DISCUSSION THREAD

**Discussion by nmanumr** [nmanumr/quizsliver#22] [Announcements]
Recently an issue was reported that the cuonline could detect the presence of the extension and doesn't let the user start the quiz. Thanks to all the reports and contributors the issue is fixed now and [v1.3.6](https://github.com/nmanumr/quizsliver/releases/tag/v1.3.6) provides the patch.

Please try out the new version and let me know if you have any feedback. Once again a special thanks to everyone who participated to publically or reached out to me privately to fix this issue.

PS: MCQs skipping issue is also fixed in v1.3.7

<hr />

The following pull requests fix the issue:
* (#21)

The following are the issues related to this fix:
* #16 
* #18

**Comment by samiraj23** [comment]
Solve This Problem plz
Failed to load extension
File
~\Downloads\quizsliver-1.3.6
Error
Manifest file is missing or unreadable
Could not load manifest.

**Reply by samiraj23** [reply]
Facing this problem while adding extension

**Reply by nmanumr** [reply]
Can you please share the contents of `~\Downloads\quizsliver-1.3.6` directory?

**Reply by Cartoonzzz** [reply]
facing the same issue.

**Reply by Cartoonzzz** [reply]
[QuizSliver.v1.3.6.zip](https://github.com/nmanumr/quizsliver/files/9794360/QuizSliver.v1.3.6.zip)

**Reply by ossamamehmood** [reply]
@samiraj23 You selected the wrong folder

### Choose Right Path

> Make Sure Load Unpacked > Select the **dist** folder inside the **quizsliver-1.3.6**

It'll work good luck

**Reply by Cartoonzzz** [reply]
ok so it is loaded. But alot of errors are showing up. 
![Screenshot (945)](https://user-images.githubusercontent.com/34788500/196034745-dd338896-0da6-4cd9-9944-437e903d4d5d.png)

**Reply by Cartoonzzz** [reply]
@nmanumr @ossamamehmood When the Quiz page is opened, it shows in green box that it is ready to start quiz, now should i take the risk and start the quiz??

**Reply by ossamamehmood** [reply]
Could you please WhatsApp me (03345463367)? 

let me take your quiz ; )

**Reply by Cartoonzzz** [reply]
check your whatsapp.

**Reply by nmanumr** [reply]
this is fixed in v1.3.7

**Reply by muhammadziyad009** [reply]
where to download 1.3.7 ? kindly guide me its only showing me 1.3.6

**Reply by ossamamehmood** [reply]
> where to download 1.3.7 ? kindly guide me its only showing me 1.3.6

Here's a newer release v1.3.7 : https://github.com/nmanumr/quizsliver/releases/tag/v.1.3.7
